Cargo.toml

```diff
@@ -16,6 +16,7 @@ repository = "https://github.com/dylanhart/ulid-rs"
 default = ["std"]
 std = ["rand"]
 postgres = ["dep:postgres-types", "dep:bytes"]
+js = ["getrandom/js"]
```

Comment:
Needs documentation in README

Reply:
Just added a line to the README. Please have a look, thanks!